static/src/js/components/GranuleFilters/__tests__/GranuleFiltersForm.test.js

```diff
@@ -761,13 +801,15 @@ describe('GranuleFiltersForm component', () => {
               }
             }
           })
+          const equatorialCrossingLatitudeSectionHeader = screen.getByText('Equatorial Crossing Longitude').parentElement.parentElement
```

Comment:
Maybe just `equatorialCrossingLatitudeSection` for these constants, since were ultimately selecting the section and not the header.

Reply:
Ah good catch I'll switch those across